zml/aio.zig

```diff
@@ -18,10 +18,10 @@ pub const log = std.log.scoped(.@"zml/aio");
 test {
     std.testing.refAllDecls(@This());
     std.testing.refAllDecls(gguf);
-    std.testing.refAllDecls(nemo);
+    // std.testing.refAllDecls(nemo);
     std.testing.refAllDecls(safetensors);
```

Comment:
I haven't ported the yaml deps to build.zig, but also not sure it's worth it. The nemo/yaml code doesn't compile anyway.